client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-main.tsx

```diff
@@ -157,79 +111,81 @@ const ScreenMain = ( {
 		elementToFocus.focus();
 	}, [ isInitialLocation ] );
 
-	useEffect( () => {
-		updateActivePatternPosition();
-	}, [ updateActivePatternPosition ] );
-
 	return (
 		<>
 			<NavigatorHeader
 				title={ <NavigatorTitle title={ translate( 'Design your own' ) } /> }
-				description={ headerDescription }
+				description={ translate(
+					'Create your homepage by first adding patterns and then choosing a color palette and font style.'
+				) }
 				hideBack
 			/>
 			<div className="screen-container__body" ref={ wrapperRef }>
 				<VStack spacing="4">
 					<NavigatorItemGroup title={ translate( 'Patterns' ) }>
 						<NavigatorItem
-							checked={ Boolean( selectedHeader ) }
+							checked={ hasHeader }
 							icon={ header }
 							aria-label={ translate( 'Header' ) }
-							onClick={ () => onMainItemSelect( { name: 'header', isPanel: true } ) }
-							active={ getIsActiveButton( 'header' ) }
+							onClick={ () => handleNavigatorItemSelect( 'header', 'header' ) }
```

Comment:
Now I'm wondering, can this be simply changed to a `<NavigatorButtonAsItem path="main/header" ...`? 😄

Reply:
Let me check the `replace` behavior. If we don't need to replace the path, we can simply use the `path` attribute.

Reply:
After retesting, I'd prefer to use the `goTo` function with the `replace: true` option since it's easy to handle the navigation. We can consider how to use the `path` to make it simple later.

Reply:
Ah, got it got it. Sure!